### PR TITLE
Bump web-app-preview

### PR DESCRIPTION
**Pull request by JanAckermann**
## Description
This bumps web-app-preview to the newest version (current master state), includes panzoom and fixes a bug where zoom with firefox didn't work, see https://github.com/owncloud/enterprise/issues/6273

## Related Issue

- Fixes https://github.com/owncloud/enterprise/issues/6273
## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Review by lookacat — APPROVED** [review]
👍🏼

File: packages/web-app-preview/package.json [hunk 1]
```diff
@@ -4,10 +4,15 @@
...
+    "web-test-helpers": "workspace:*"
+  },
   "peerDependencies": {
-    "vue3-gettext": "^2.3.3",
-    "vuex": "4.1.0",
     "web-client": "workspace:@ownclouders/web-client@*",
-    "web-pkg": "workspace:@ownclouders/web-pkg@*"
+    "web-pkg": "workspace:@ownclouders/web-pkg@*",
+    "@vueuse/core": "^10.3.0",
+    "vue3-gettext": "2.4.0",
```
Comment (JammingBen) [inline packages/web-app-preview/package.json:15]: I feel like we shouldn't bump any dependencies if not necessary in a patch release.

```suggestion
    "vue3-gettext": "^2.3.3",
```

File: packages/web-app-preview/src/components/Sources/MediaImage.vue [hunk 2]
```diff
@@ -26,6 +29,63 @@ export default defineComponent({
       type: Number,
       required: true
     }
+  },
+  setup(props) {
+    const img = ref()
```
Comment (kulmann) [inline packages/web-app-preview/src/components/Sources/MediaImage.vue:34]: Could you update this again from current master? @JammingBen and myself changed this today.

Comment (JanAckermann) [reply packages/web-app-preview/src/components/Sources/MediaImage.vue:34]: Sure

**Review by kulmann — APPROVED** [review]
nice, thank you 💪

**Pull request merged**